lib/cylc/gui/updater_dot.py

```diff
@@ -181,7 +181,7 @@ def set_led_headings(self):
         if len(self.family_tree) > 0:
```

Comment:
`if self.family_tree:`?

Reply:
Outside of the scope of pylint changes but I'll change it anyway.